core/src/main/java/org/hisp/dhis/android/core/trackedentity/internal/TrackedEntityInstanceFilterCall.kt

```diff
@@ -29,49 +29,32 @@ package org.hisp.dhis.android.core.trackedentity.internal
 
 import org.hisp.dhis.android.core.arch.api.executors.internal.APIDownloader
 import org.hisp.dhis.android.core.arch.call.factories.internal.UidsCallCoroutines
-import org.hisp.dhis.android.core.common.internal.DataAccessFields
 import org.hisp.dhis.android.core.systeminfo.DHISVersion
 import org.hisp.dhis.android.core.systeminfo.DHISVersionManager
 import org.hisp.dhis.android.core.trackedentity.TrackedEntityInstanceFilter
 import org.koin.core.annotation.Singleton
 
 @Singleton
 class TrackedEntityInstanceFilterCall internal constructor(
-    private val service: TrackedEntityInstanceFilterService,
+    private val networkHandler: TrackedEntityInstanceFilterNetworkHandler,
     private val handler: TrackedEntityInstanceFilterHandler,
     private val apiDownloader: APIDownloader,
     private val versionManager: DHISVersionManager,
 ) : UidsCallCoroutines<TrackedEntityInstanceFilter> {
     override suspend fun download(uids: Set<String>): List<TrackedEntityInstanceFilter> {
-        val accessDataReadFilter = "access." + DataAccessFields.read.eq(true).generateString()
-
         return if (versionManager.isGreaterOrEqualThan(DHISVersion.V2_38)) {
```

Comment:
In VisualizationCall, this logic was moved to the network layer, I think we talked about that and agreed on it. I don't have a strong opinion, but just to be consistent